ToolChain/Debugger/Debugger.py

```diff
@@ -97,11 +99,38 @@ def __init__(self, inputFile=None,
             self.symbols = {}
             self.loadSymbols(symbolsFile=symbolsFile)
 
+        if breakpointFile != "" and breakpointFile is not None:
```

Comment:
The first part of the check should be done in the calling function at the same time it checks if the path exists.